### PR TITLE
ADEN-13135 NTC 2.0 + Metacritic events + first-second call on News&Ratings

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Comment by dmnsobczak** [comment]
Merged PRs:
- https://github.com/Wikia/ad-engine/pull/1897

File: src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-neutron-helper.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+			},
+			false,
+		);
+	}
+
+	setupPageChangedWatcher(container: Container, ...steps: any[]) {
+		communicationService.on(
+			eventsRepository.PLATFORM_PAGE_CHANGED,
+			() => {
+				if (this.firstPageview) {
```
Comment (preludek) [inline src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-neutron-helper.service.ts:37]: 😞

**Review by preludek — DISMISSED** [review]
Looks good

File: src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-neutron-helper.service.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/platforms/news-and-ratings/shared/setup/dynamic-slots/news-and-ratings-neutron-helper.service.ts]: Awaiting fix :(
https://fandom.atlassian.net/browse/ADEN-13243

**Review by kwfandom — APPROVED** [review]

**Pull request merged**